great_expectations/core/expectation_suite.py

```diff
@@ -562,7 +565,8 @@ def find_expectation_indexes(
                     match_indexes.append(idx)
             else:  # noqa: PLR5501
                 if expectation.configuration.isEquivalentTo(
-                    other=expectation_configuration, match_type=match_type  # type: ignore[arg-type]
+                    other=expectation_configuration,
+                    match_type=match_type,  # type: ignore[arg-type]
                 ):
```

Comment:
i'm confused - why did this change?

Reply:
ah, looks like you can just remove this type ignore

Reply:
yeah, looks like autoformat moved it and it can go 🎉